tests/Go/Functional/BaseFunctionalTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Go\Functional;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Process;
+
+abstract class BaseFunctionalTest extends TestCase
```

Comment:
When you find a time, please review tests as well.

I have cleaned up code a little bit, setup a base for functional tests, write a required test as well as modified (cleaned up) command tests...

I have one more thing on my mind, however, even with this setup it is going to be quite easy to write a bunch of functional tests that we need.